jest.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     },
     globals: {
         CONFIG: {
-            API_HOST: '',
+            API_HOST: 'http://localhost',
```

Comment:
Should this have a port?!

Reply:
this is just for the tests, so not really necessary